var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -562,13 +572,14 @@ def determine_b2_options(self, spec, options):
 
         options.extend(["link=%s" % ",".join(link_types), "--layout=%s" % layout])
 
-        if not spec.satisfies("@:1.75 %intel"):
+        if not (spec.satisfies("@:1.75 %intel") or spec.satisfies("platform=windows")):
             # When building any version >= 1.76, the toolset must be specified.
             # Earlier versions could not specify Intel as the toolset
             # as that was considered to be redundant/conflicting with
             # --with-toolset in bootstrap.
             # (although it is not currently known if 1.76 is the earliest
             # version that requires specifying the toolset for Intel)
+            # And again, with no cl.exe/ifx wrapper this will fail on Windows
```

Comment:
Avoid using the term "again": it makes sense within the context of the PR, but once the changes get merged into the file, it provides no real context.